### ISSUE TITLE
Can we add breakpoint/restart feature?

### ISSUE THREAD

**Issue by vigo332** [saalfeldlab/stitching-spark#9]
Since the running job takes long time to finish for our workflow, it would be nice to dump generic breakpoints or restart points periodically so that we can start from the latest restart points in case of mishaps. I have my analysis killed after 15 hours of running/or failed due to oom, it is a pain to restart it from the beginning with new configurations. Thanks.

**Comment by igorpisarev** [comment]
I agree that persisting partial results would be a good thing to have. I'll put it on my to-do list. Thanks for the suggestion!
